ArduPlane/qautotune.h

```diff
@@ -8,7 +8,7 @@
 
 #include "quadplane.h"
 #ifndef QAUTOTUNE_ENABLED
-  #define QAUTOTUNE_ENABLED HAL_QUADPLANE_ENABLED
+  #define QAUTOTUNE_ENABLED CONFIG_HAL_BOARD == HAL_BOARD_SITL
```

Comment:
Why this change?

Reply:
i want to remove QAUTOTUNE mode by default, it is just too dangerous for most users. There are some aircraft it works for, but far more often it causes a crash, plus the aircraft needs a very long hover time.
This saves us a lot of flash space which gives us room for quicktune as C++ which does actually work